src/api/ParticipantManager.API/Models/PathwayTypeEnrolment.cs

```diff
@@ -20,7 +20,7 @@ public class PathwayTypeEnrolment
 
     public DateTime? NextActionDate { get; set; }
 
-    public required Participant Participant { get; set; }
+    public Participant? Participant { get; set; }
```

Comment:
This change unnerves me, but then I'm out of practice with EF.

Remind me - is this so that we can create a PathwayTypeEnrolment without having to reference the whole associated Participant? And, conversely, so we could choose to query for PathwayTypeEnrolments without eagerly loading the linked Participants?